README.md

```diff
@@ -1158,50 +1158,50 @@ _You can enable the following settings in Xcode by running [this script](resourc
   ```swift
   // WRONG
   if let galaxy {
-    …
+    ...
```

Comment:
Sometimes we use this style and sometimes we put the `...` in a comment.

https://github.com/airbnb/swift/blob/742bf391c328025f262501ed13493c927af8eb80/README.md?plain=1#L118-L120

I feel like we should consolidate on one style... Maybe we update the guide to convert `// ...` to `...`?

I have used the `// ...` pattern in the past, but since `...` isn't valid Swift, maybe it's better to use the simpler style, especially since it works for single-line blocks too.

What do you think @calda ? It's also okay if we tackle this in a future PR.